lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.ts

```diff
@@ -27,7 +27,9 @@ import { takeUntil, map } from 'rxjs/operators';
 @Component({
     selector: 'adf-card-view-selectitem',
     templateUrl: './card-view-selectitem.component.html',
-    styleUrls: ['./card-view-selectitem.component.scss']
+    styleUrls: ['./card-view-selectitem.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    host: { class: 'adf-card-view-selectitem' }
```

Comment:
should not be select-item instead of selectitem? You can also rename component's selector and files names as they seem have the same typo

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379735582 comment as similar case